### ISSUE TITLE
Set the header parameter X-Watson-Learning-Opt-Out to true

### ISSUE THREAD

**Issue by a-ro** [watson-developer-cloud/python-sdk#83]
I'm working with private documents and need to set X-Watson-Learning-Opt-Out to true for any watson services I'm using on these documents. There doesn't seem to be a way to do this currently with the python-sdk.  One way to solve this would be to add a parameter x_watson_learning_opt_out in WatsonDeveloperCloudService and verify this parameter in request to set the header.
```
class WatsonDeveloperCloudService(object):
    def __init__(self, vcap_services_name, url, username=None, password=None, use_vcap_services=True, api_key=None, x_watson_learning_opt_out=False):
        self.x_watson_learning_opt_out = x_watson_learning_opt_out

    def request( ....)
        ....
        if self.x_watson_learning_opt_out:
            headers['X-Watson-Learning-Opt-Out'] = True
```

**Comment by jsstylos** [comment]
Thanks, added in 65c95783a373485107f7cd6aefdb53b8491710ba

**Comment by a-ro** [comment]
Thanks for the quick fix, but you really need to set headers['X-Watson-Learning-Opt-Out'] = **true** to disable the logging (instead of false in your code).

**Comment by jsstylos** [comment]
D'oh! Good catch. Fixed (again) in ec057cd582c5957238c6da9be0906d2fce551e06
